src/wallet/api/wallet_manager.cpp

```diff
@@ -91,7 +97,7 @@ std::vector<std::string> WalletManagerImpl::findWallets(const std::string &path)
     boost::filesystem::path work_dir(path);
     // return empty result if path doesn't exist
     if(!boost::filesystem::is_directory(path)){
-      return result;
+        return result;
```

Comment:
gratuitous whitespace changes are frowned upon (unless the previous state is dire)